mmv1/third_party/terraform/services/container/resource_container_node_pool_test.go.tmpl

```diff
@@ -6080,3 +6080,197 @@ resource "google_container_node_pool" "np" {
 `, cluster, np, networkName, subnetworkName, storagePoolResourceName, location)
 }
 
+func testAccContainerNodePool_acceleratorNetworkProfile(clusterName, npName string) string {
+	return fmt.Sprintf(`
+resource "google_container_cluster" "cluster" {
+  name     = "%s"
+  location = "us-central1-a"
+
+  datapath_provider       = "ADVANCED_DATAPATH"
+  ip_allocation_policy    {} // Enables IP aliasing
+  enable_multi_networking = true
+
+  initial_node_count = 1
+  deletion_protection = false
+}
+
+resource "google_container_node_pool" "np" {
+  name     = "%s"
+  cluster  = google_container_cluster.cluster.id
+  location = "us-central1-a"
+  node_count = 1
+
+  node_config {
+    machine_type = "a3-highgpu-8g"
```

Comment:
you mentioned stockout issues related to this machine type; is there a more available machine type we could use instead?